Tutorials/start/telemetrystart.md

```diff
@@ -1,16 +1,36 @@
+Since [Telemetry](../../Applications/Telemetry) responds with JSON data we could use a web browser to print it to the screen, Python, or any application able to query JSON responses with HTTP communications to interact.
+
+## Initializing Telemetry
+Just as we've done many times, we need to initialize the  `faraday-telemetry` configuration file. Simply run:
+* `faraday-telemetry --init-config`
+
+![Telemetry Configuration](images/faraday-telemetry-init.jpg)
 
 ## Configuring & Running Telemetry
 
-The configuration guide in [Telemetry](../../Applications/Telemetry) documentation sums up how to get the application up and running. We suggest you head over to that documentation to step through configuration. Once setup, play around with the example queries using your web browser located on the bottom of the documentation. Report back here when complete!
+To configure the `faraday-telemetry` application simply tell it what the Proxy callsign and nodeid is as programmed when we configured `faraday-proxy`:
+
+* `faraday-telemetry --callsign kb1lqc --nodeid 2`
+
+Running this command will start the Telemetry application which should be left running.
+
+![Faraday Telemetry](images/faraday-telemetry.jpg)
+
+## Querying The Telemetry Application
+
+`faraday-telemetry` uses a RESTful API to communicate over HTTP requests. However, you can easily use your web browser to interact. When you are querying the [Telemetry](../../Applications/Telemetry) application you will see data printed directly to your web browser. Reloading the webpage will perform the query again and update the information on the screen. This is a handy way to quickly debug applications as you develop them!
 
-When you are querying the [Telemetry](../../Applications/Telemetry) application you will see data printed directly to your web browser. Reloading the webpage will perform the query again and update the information on the screen. This is a handy way to quickly debug applications as you develop them!
+Example queries
+* See all telemetry data (up to 1000 items): [http://localhost:8001](http://localhost:8001)
+* See last five telemetry items: [http://localhost:8001?limit=5](http://localhost:8001?limit=5)
+* See telemetry from specific station: [http://localhost:8001/?callsign=kb1lqc&nodeid=2](http://localhost:8001/?callsign=kb1lqc&nodeid=2)
+* See stations heard in last 5 minutes: [http://localhost:8001/stations](http://localhost:8001/stations)
 
 ![Telemetry output in web browser](images/WebBrowser_FullOutput.exe.png)
 
 # Moving Forward
-Now that you are obtaining telemetry from your Faraday radio it is time to command it. Turn on the LED in our [Hello World](hello-world.md) guide!
\ No newline at end of file
+Now that you are obtaining telemetry from your Faraday radio it is time to command it. Turn on the LED in our [Hello World](hello-world.md) guide!
```

Comment:
ACTION @kb1lqc : You may want to mention where the new telemetry files are stored (in the USERS .faraday/lib folder):

![image](https://user-images.githubusercontent.com/339144/27819774-98b95938-6050-11e7-8772-b0371b72efad.png)